cmd/kubelet/app/options/options.go

```diff
@@ -29,6 +30,26 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	defaultRootDir             = "/var/lib/kubelet"
+	experimentalFlannelOverlay = false
+
+	// When these values are updated, also update test/e2e/framework/util.go
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "3.0"
+	// Auto detect cloud provider.
+	AutoDetectCloudProvider         = "auto-detect"
+	DefaultKubeletPodsDirName       = "pods"
+	DefaultKubeletVolumesDirName    = "volumes"
+	DefaultKubeletPluginsDirName    = "plugins"
+	DefaultKubeletContainersDirName = "containers"
```

Comment:
@mtaufen @saad-ali 
Just want to confirm that those default values are only used for kubelet internally. So options.go is the appropriate place for those values?